draft-ietf-quic-transport.md

```diff
@@ -1194,13 +1194,14 @@ server then needs to validate that the client owns the address that it claims.
 QUIC therefore performs source address validation during connection
 establishment.
 
-Servers MUST NOT send more than three times as many bytes as the number of bytes
-received prior to verifying the client's address.  Source addresses can be
+Prior to verifying the client's address, servers MUST NOT send more than three
```

Comment:
Q: Is this actually less awkward?  It's ok, but seems neutral to me.  Both sentences are a bit odd.

Reply:
"prior to verifying the client's address" could be construed as modifying "bytes received". When I remove all context for what I know about this, that's how I read the sentence. A new reader would have to reason a bit to understand what you mean, IMO.

if you're neutral I'll stick with it.